Intel Low Power Subsystem Support
Select to build support for Intel Low Power Subsystem such as
found on Intel Lynxpoint PCH. Selecting this option enables
things like clock tree (common clock framework) and pincontrol
which are needed by the LPSS peripheral drivers.
Depends: X86 && ACPI && PCI
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:19:24.854610365 -0400
@@ -340,7 +340,7 @@
 CONFIG_X86_UV=y
 # CONFIG_X86_GOLDFISH is not set
 # CONFIG_X86_INTEL_MID is not set
-CONFIG_X86_INTEL_LPSS=y
+# CONFIG_X86_INTEL_LPSS is not set
 CONFIG_X86_AMD_PLATFORM_DEVICE=y
 CONFIG_IOSF_MBI=y
 # CONFIG_IOSF_MBI_DEBUG is not set
@@ -6656,21 +6656,6 @@
 CONFIG_SND_SOC_INTEL_SOF_MAXIM_COMMON=m
 CONFIG_SND_SOC_INTEL_SOF_REALTEK_COMMON=m
 CONFIG_SND_SOC_INTEL_SOF_CIRRUS_COMMON=m
-CONFIG_SND_SOC_INTEL_HASWELL_MACH=m
-CONFIG_SND_SOC_INTEL_BDW_RT5650_MACH=m
-CONFIG_SND_SOC_INTEL_BDW_RT5677_MACH=m
-CONFIG_SND_SOC_INTEL_BROADWELL_MACH=m
-CONFIG_SND_SOC_INTEL_BYTCR_RT5640_MACH=m
-CONFIG_SND_SOC_INTEL_BYTCR_RT5651_MACH=m
-CONFIG_SND_SOC_INTEL_BYTCR_WM5102_MACH=m
-CONFIG_SND_SOC_INTEL_CHT_BSW_RT5672_MACH=m
-CONFIG_SND_SOC_INTEL_CHT_BSW_RT5645_MACH=m
-CONFIG_SND_SOC_INTEL_CHT_BSW_MAX98090_TI_MACH=m
-CONFIG_SND_SOC_INTEL_CHT_BSW_NAU8824_MACH=m
-CONFIG_SND_SOC_INTEL_BYT_CHT_CX2072X_MACH=m
-CONFIG_SND_SOC_INTEL_BYT_CHT_DA7213_MACH=m
-CONFIG_SND_SOC_INTEL_BYT_CHT_ES8316_MACH=m
-CONFIG_SND_SOC_INTEL_BYT_CHT_NOCODEC_MACH=m
 CONFIG_SND_SOC_INTEL_SKL_RT286_MACH=m
 CONFIG_SND_SOC_INTEL_SKL_NAU88L25_SSM4567_MACH=m
 CONFIG_SND_SOC_INTEL_SKL_NAU88L25_MAX98357A_MACH=m
@@ -6756,7 +6741,6 @@
 #
 # CODEC drivers
 #
-CONFIG_SND_SOC_ARIZONA=m
 CONFIG_SND_SOC_WM_ADSP=m
 CONFIG_SND_SOC_AC97_CODEC=m
 CONFIG_SND_SOC_ADAU_UTILS=m
@@ -6829,7 +6813,6 @@
 # CONFIG_SND_SOC_ICS43432 is not set
 # CONFIG_SND_SOC_INNO_RK3036 is not set
 CONFIG_SND_SOC_MAX98088=m
-CONFIG_SND_SOC_MAX98090=m
 CONFIG_SND_SOC_MAX98357A=m
 # CONFIG_SND_SOC_MAX98504 is not set
 CONFIG_SND_SOC_MAX9867=m
@@ -6876,13 +6859,9 @@
 # CONFIG_SND_SOC_RT5631 is not set
 CONFIG_SND_SOC_RT5640=m
 CONFIG_SND_SOC_RT5645=m
-CONFIG_SND_SOC_RT5651=m
 CONFIG_SND_SOC_RT5659=m
 CONFIG_SND_SOC_RT5660=m
 CONFIG_SND_SOC_RT5663=m
-CONFIG_SND_SOC_RT5670=m
-CONFIG_SND_SOC_RT5677=m
-CONFIG_SND_SOC_RT5677_SPI=m
 CONFIG_SND_SOC_RT5682=m
 CONFIG_SND_SOC_RT5682_I2C=m
 CONFIG_SND_SOC_RT5682_SDW=m
@@ -6938,7 +6917,6 @@
 # CONFIG_SND_SOC_TSCS454 is not set
 # CONFIG_SND_SOC_UDA1334 is not set
 # CONFIG_SND_SOC_WCD938X_SDW is not set
-CONFIG_SND_SOC_WM5102=m
 # CONFIG_SND_SOC_WM8510 is not set
 # CONFIG_SND_SOC_WM8523 is not set
 CONFIG_SND_SOC_WM8524=m
